manifests/params.pp

```diff
@@ -33,6 +33,35 @@
         ],
         'maxconn' => '8000'
       }
+        $config_file      = '/etc/haproxy/haproxy.cfg'
+      }
+    'FreeBSD': {
+      $package_name     = 'haproxy'
+      $global_options   = {
+        'log'           => [
+          '127.0.0.1 local0',
+          '127.0.0.1 local1 notice',
+        ],
+        'chroot'        => '/usr/share/haproxy',
```

Comment:
I wasn't, but thats a good idea.  I didn't see where the logs were defined, but I'm not very familiar with the haproxy config.